core/common/src/main/java/alluxio/worker/block/BlockStoreLocation.java

```diff
@@ -154,7 +172,7 @@ public String mediumType() {
    * @return true if location is {@link #ANY_MEDIUM}
```

Comment:
modify the java doc to match the new logic?

Reply:
The logic is the same. `ANY_MEDIUM` is an empty string.

Reply:
got it, sounds good to me